src/Symfony/Bridge/Monolog/Logger.php

```diff
@@ -70,8 +70,12 @@ public function clear()
     /**
      * {@inheritdoc}
      */
-    public function reset()
+    public function reset(): void
     {
+        if (\is_callable(array(parent::class, 'reset'))) {
```

Comment:
`if ($this instance of Monolog\ResettableInterface) {`?

Reply:
I was obviously thinking too complicated. Good catch. 😃